parsl/dataflow/dflow.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, config=Config()):
         workflow_info = {
                 'python_version': sys.version_info,
                 'parsl_version': get_version(),
-                "time_began": str(self.time_began.strftime('%Y-%m-%d %H:%M:%S')),
+                "time_began": str(self.time_began),
```

Comment:
Can we make float columns be `sqlalchemy.Float` type? Then we don't need to convert this to a string.

Reply:
Currently, all log information is being converted to strings by the handler that sends the log over HTTP inside db_local.py RemoteHandler. I convert them to strings here to be explicit about this. The reason for this conversion had to do with a problem while sending the information over HTTP.

Reply:
Got it, thanks for the clarification!